doc/tutorials/klondike/app/lib/step2/klondike_game.dart

```diff
@@ -0,0 +1,50 @@
+import 'package:flame/components.dart';
+import 'package:flame/game.dart';
+import 'package:flame/experimental.dart';
+
+import 'components/foundation.dart';
+import 'components/pile.dart';
+import 'components/stock.dart';
+import 'components/waste.dart';
+
+class KlondikeGame extends FlameGame {
+  final gap = 175.0;
+  final w = 1000.0;
+  final h = 1400.0;
+
+  @override
+  Future<void> onLoad() async {
+    await images.load('klondike-sprites.png');
+
+    final stock = Stock()
+      ..size = Vector2(w, h)
+      ..position = Vector2(gap, gap);
+    final waste = Waste()
+      ..size = Vector2(w * 1.5, h)
+      ..position = Vector2(w + 2*gap, gap);
+    final foundations = [
+      for (var i = 0; i < 4; i++)
+        Foundation()
+          ..size = Vector2(w, h)
+          ..position = Vector2((i + 3) * (w + gap) + gap, gap)
+    ];
+    final piles = [
+      for (var i = 0; i < 7; i++)
+        Pile()
+          ..size = Vector2(w, h)
+          ..position = Vector2(gap + i * (w + gap), h + 2*gap)
+    ];
+
+    final world = World()..addToParent(this);
```

Comment:
Might be clearer to split this up on two lines and do `add(world);`?

Reply:
I was trying to demonstrate here that there are different ways of adding components to the game, there is a comment about this in the doc.

Reply:
I feel that this one is kind of hard to read though and I don't think we should encourage the `addToParent(tihs)` pattern, when `add(component)` is easier to read? But would be good for sure to show how `addToParent` is used too like you say, but maybe somewhere where it's not already in the component that it is being added to?

Reply:
Alright, I changed it into simple `add()`.